modules/nf-core/pilon/tests/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process PILON"
+    script "../main.nf"
+    process "PILON"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pilon"
+
+
+    test("Test PILON") {
```

Comment:
Change test name to something more specific, there is no standard (yet) but something like `pilon - bam - paired` should do